sale_order_invoicing_queued/readme/INSTALL.md

```diff
@@ -1,6 +0,0 @@
-This module depends on *queue_job* module that is hosted on
```

Comment:
The first part of this text is OK to be kept. It's just the second paragraph which should be deleted.